src/utils/verifyAuthToken.ts

```diff
@@ -8,8 +12,15 @@ import jwt from "@tsndr/cloudflare-worker-jwt";
  */
 
 export async function verifyAuthToken(authHeader: string, env: env) {
-  const authToken = authHeader.split(" ")[1];
-  await jwt.verify(authToken, env.RDS_SERVERLESS_PUBLIC_KEY, {
+  const parts = authHeader.split(" ");
+  if (parts.length !== 2 || parts[0] !== "Bearer") {
+    throw new Error(INVALID_TOKEN_FORMAT);
+  }
+  const authToken = parts[1];
+  const isValid = await jwt.verify(authToken, env.RDS_SERVERLESS_PUBLIC_KEY, {
     algorithm: "RS256",
   });
+  if (!isValid) {
+    throw new Error(AUTHENTICATION_ERROR);
+  }
```

Comment:
Won't it throw an error by default if verify fails?

Reply:
Library's documentation specify that it will return a boolean based on the verification result.
![Screenshot 2023-12-25 at 12 48 35 AM](https://github.com/Real-Dev-Squad/discord-slash-commands/assets/98796547/7f77e8ed-bc04-4ad9-8494-ad22a58fd82f)
https://github.com/tsndr/cloudflare-worker-jwt